pypeit/spectrographs/spectrograph.py

```diff
@@ -1140,10 +1152,13 @@ def get_meta_value(self, inp, meta_key, required=False,
                             if ftype in self.meta[meta_key]['required_ftypes']:
                                 kerror = True
                     # Bomb out?
+                    # TODO Bombing out is not acceptable as we have discussed.
                     if kerror:
                         #embed(header=utils.embed_header())
-                        msgs.error('Required meta "{0}" did not load!'.format(meta_key)
-                                   + 'You may have a corrupt header.')
+                        msgs.warn('Required meta "{0}" did not load!'.format(meta_key)
```

Comment:
Put this back as an `error`.  It needs to bomb out in a standard `run_pypeit` call.

Reply:
This causes crashes with bad headers. This was the only way to fix it. The logic in this routine is impossible to understand and it needs to be refactored entirely. I'm going to leave it as is for now, since I really don't like it when one bad header file causes pypeit_setup to crash. Again, it never should since then the user is dead in the water.

Reply:
Am betting my PR to this branch will fix this.
But am not going to let this poison pill change to through here.

It is not even related to the topic of the PR (`objfind_refactor`)